xrviz/projection.py

```diff
@@ -87,6 +87,15 @@ def setup(self, *args):
                 widget.disabled = disabled
         self.proj_params.disabled = disabled
 
+    def setup_initial_values(self, init_params):
+        is_geo_disabled = self.is_geo.disabled  # to restore state of is_geo
+        if init_params:
+            self.is_geo.disabled = False
```

Comment:
This pattern has come up a few times, why set and reset the value?
Simpler:
```
def setup_initial_values(self, init_params={}):
    for widget in [self.projection, self.proj_params]:   # or use strings here
        if widget.name in init_params:
            widget.value = init_params[widget.name]
```
I wasn't totally sure whether you wanted is_geo in this list or not. The simplest way not to automatically set it, is simply not to pass it in the init_params

Reply:
I have made modifications according to this method. Thanks!